assembly-syntax/src/parser/token.rs

```diff
@@ -204,11 +204,6 @@ pub enum Token<'input> {
     PushMapval,
     PushMapvaln,
     PushMtnode,
-    PushSmtpeek,
-    PushSmtset,
-    PushSmtget,
```

Comment:
I couldn't find anywhere this was used, is it deprecated now that these are in `stdlib::collections::smt`?